ethereum/statetransition/src/main/java/tech/pegasys/teku/statetransition/block/BlockImportChannel.java

```diff
@@ -13,11 +13,20 @@
 
 package tech.pegasys.teku.statetransition.block;
 
+import java.util.Optional;
 import tech.pegasys.teku.infrastructure.async.SafeFuture;
 import tech.pegasys.teku.infrastructure.events.ChannelInterface;
 import tech.pegasys.teku.spec.datastructures.blocks.SignedBeaconBlock;
 import tech.pegasys.teku.spec.logic.common.statetransition.results.BlockImportResult;
 
 public interface BlockImportChannel extends ChannelInterface {
-  SafeFuture<BlockImportResult> importBlock(SignedBeaconBlock block);
```

Comment:
Could possibly leave this as default and pass Optional.empty()?

Reply:
I thought about that but having a default implementation essentially used for testing, i remember @zilm13 complaining about that. but for sure it will reduce the noise.

Reply:
I guess the work is done now, and it does kinda make it cleaner to have the 1 interface to support..